src/pages/saksbehandling/revurdering/bosituasjon/BosituasjonForm.tsx

```diff
@@ -0,0 +1,435 @@
+                            onAlderChange={(alder) => setEPSAlder(alder)}
+                        />
+                    )}
+                />
+                {epsAlder && epsAlder < 67 && (
+                    <Controller
+                        control={form.control}
+                        name="erEPSUførFlyktning"
+                        render={({ field, fieldState }) => (
+                            <RadioGruppe
+                                legend={intl.formatMessage({ id: 'form.erEPSUførFlyktning' })}
+                                feil={fieldState.error?.message}
+                            >
+                                <Radio
+                                    label="Ja"
+                                    name="erEPSUførFlyktning"
+                                    checked={field.value === true}
+                                    onChange={() => field.onChange(true)}
+                                />
+                                <Radio
+                                    label="Nei"
+                                    name="erEPSUførFlyktning"
+                                    checked={field.value === false}
+                                    onChange={() => field.onChange(false)}
+                                />
+                            </RadioGruppe>
+                        )}
+                    />
+                )}
+            </div>
+        );
+    };
+
+    function resetEPSForm() {
+        form.setValue('epsFnr', null);
+        form.setValue('erEPSUførFlyktning', null);
+    }
+
+    function resetDelerSøkerBoligForm() {
+        form.setValue('delerSøkerBolig', null);
+    }
+
+    return (
+        <ToKolonner tittel={<RevurderingsperiodeHeader periode={props.revurdering.periode} />}>
+            {{
+                left: (
+                    <form className={sharedStyles.revurderingContainer} onSubmit={form.handleSubmit(handleSubmit)}>
+                        <div>
+                            <div className={styles.bosituasjonInputsContainer}>
+                                <Controller
+                                    control={form.control}
+                                    name="harEPS"
+                                    render={({ field, fieldState }) => (
+                                        <RadioGruppe
+                                            legend={intl.formatMessage({ id: 'form.harSøkerEPS' })}
+                                            feil={fieldState.error?.message}
+                                        >
+                                            <Radio
+                                                label="Ja"
+                                                name="harEPS"
+                                                checked={field.value === true}
+                                                onChange={() => {
+                                                    resetDelerSøkerBoligForm();
```

Comment:
Kunne dette heller ha blitt gjord ved å watche `harEps`-feltet i formet?

Reply:
Tror ikke jeg helt henger med

Reply:
I stedet for å gjøre dette inni onchange-en her; kunne vi ha hatt noe ala

```ts
const harEps = form.watch('harEPS');
useEffect(() => {
    if (harEps === null) {
        return;
    }
    if (harEps) {
        resetDelerSøkerBoligForm();
    } else {
        resetEPSForm();
    }
}, [harEps]);
```